tests/basic_fuzzer.cpp

```diff
@@ -18,62 +28,361 @@ namespace {
 std::array<size_t, 10> input_size{7, 16, 12, 64, 67, 128, 129, 256, 1024, 1025};
```

Comment:
Maybe we want to allow arbitrary, randomly selected input sizes?

Reply:
Yes I can do that, random input sizes varying in some range.

Reply:
I think we may want the input sizes to be randomly generated.